mirrord/kube/src/api/container/job.rs

```diff
@@ -152,6 +152,8 @@ where
                 "disabled".to_string(),
             ),
             ("app".to_string(), "mirrord".to_string()),
+            ("prometheus.io/scrape".to_string(), "true".to_string()),
+            ("prometheus.io/port".to_string(), "9000".to_string()),
```

Comment:
These labels may vary between scrapers, no? I think we should let the user set them via agent config (just mention port number and endpoint path in the doc)

Reply:
Yeah, was there for convenience while I was trying out stuff, removed in favour of config.